action/alarmWebAction.js

```diff
@@ -131,7 +131,8 @@ function main(params) {
         return new Promise(function (resolve, reject) {
             common.verifyTriggerAuth(triggerData, false)
             .then(() => {
-                db = new Database(params.DB_URL, params.DB_NAME);
+                db = getDatabase(params.DB_URL, params.DB_NAME, params.DB_TYPE,
+                    params.COSMOSDB_ROOT_DB, params.COSMOSDB_MASTERKEY);
```

Comment:
instead of "all possible params" being in the sig, I think it might be simpler to pass a config object, which will vary based on db type, e.g.
```
let cosmosConfig = {type: "cosmosdb", url: "", rootdb:"", masterkey:""};
let couchConfig = {type: "couchdb", url: "", dbname:""};
getDatabase(config);
```